libs/wire-api/src/Wire/API/User/Client/Prekey.hs

```diff
@@ -37,17 +37,30 @@ module Wire.API.User.Client.Prekey
   )
 where
 
+import Control.Lens ((?~))
 import Data.Aeson
+import Data.Data (Proxy (Proxy))
 import Data.Hashable (hash)
 import Data.Id
+import Data.Swagger (HasDescription (description), HasSchema (schema), ToSchema (..))
 import qualified Data.Swagger.Build.Api as Doc
+import Deriving.Swagger (CustomSwagger (..), FieldLabelModifier, LabelMapping ((:->)), LabelMappings, LowerCase, StripPrefix)
 import Imports
 import Wire.API.Arbitrary (Arbitrary (arbitrary), GenericUniform (..))
 
 newtype PrekeyId = PrekeyId {keyId :: Word16}
   deriving stock (Eq, Ord, Show, Generic)
   deriving newtype (ToJSON, FromJSON, Arbitrary)
 
+instance ToSchema PrekeyId where
+  declareNamedSchema _ = tweak $ declareNamedSchema (Proxy @Int)
+    where
+      tweak = fmap $ schema . description ?~ descr
+        where
+          descr = "in the range [0..65535]."
+
+-- FUTUREWORK: can this be also expressed in swagger, not just in the description?
+
```

Comment:
If you newtype derive ToSchema, it will have the right bounds in the schema by default.